schemas/Claim.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Claim.json",
+    "title": "Claim",
+    "description": "A claim for a future economic event(s) in reciprocity for an economic event that already occurred. For example, a claim for payment for goods received.",
+    "type": "object",
+    "properties": {
+        "action": {
+            "description": "Relates a process input or output to a verb, such as consume, produce, work, improve, etc.",
+            "type": "string"
+        },
+        "resourceQuantity": {
+            "description": "The amount and unit of the economic resource counted or inventoried.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/QuantityValue.json"
+        },
+        "effortQuantity": {
+            "description": "The amount and unit of the work or use or citation effort-based action. This is often a time duration, but also could be cycle counts or other measures of effort or usefulness.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/QuantityValue.json"
+        },
+        "resourceClassifiedAs": {
```

Comment:
Why does a claim have a resource? I looked at NRP, our claims do not have resources.

Reply:
not a resource, just classification or specification, should have that, right?  if you are claiming a type of resource